lib/index.js

```diff
@@ -104,5 +140,4 @@ export default class Pageres {
 	}
 }
 
-objectAssign(Pageres.prototype, EventEmitter.prototype);
-objectAssign(Pageres.prototype, require('./util'));
+Object.assign(Pageres.prototype, util);
```

Comment:
I ended up just moving everything into the main file. Didn't really make sense as a mixing. Will split out some into real utility methods later on.